src/Tool/MacAuthorizationTrait.php

```diff
@@ -26,7 +26,7 @@
     /**
      * Returns the id of this token for MAC generation.
      *
-     * @param  AccessToken $token
+     * @param  AccessToken|null $token
```

Comment:
Type hint does not allow the token to be `null` and the method clearly needs a valid token.